src/constants/constants.ts

```diff
@@ -77,6 +77,7 @@ export const REDIRECTION_SERVER_IPS = [
   "18.139.47.66",
 ]
 export const DNS_INDIRECTION_DOMAIN = "hostedon.isomer.gov.sg"
+export const DNS_CNAME_SUFFIXES = ["cloudfront.net", "kxcdn.com"]
```

Comment:
sir I meant that its declared in the constants file here haha

Reply:
Ah okay, updated in 3ea7d34d8fbb597b2b1826cff233f5b856aa8a43!